scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/internal/propertyAccessors.scala

```diff
@@ -31,7 +33,8 @@ object propertyAccessors {
       new NoParamMethodPropertyAccessor(accessChecker),
       // it can add performance overhead so it will be better to keep it on the bottom
       MapLikePropertyAccessor,
-      MapMissingPropertyToNullAccessor, // must be after NoParamMethodPropertyAccessor
+      new ExtensionMethodsPropertyAccessor(accessChecker, set),
```

Comment:
nip: Maybe let's move it after NoParamMethodPropertyAccessor to keep map related accessors together

Reply:
Done